presto-hive/src/test/java/com/facebook/presto/hive/TestJsonHiveHandles.java

```diff
@@ -42,7 +43,7 @@
             .put("hiveType", "float")
             .put("typeSignature", "double")
             .put("hiveColumnIndex", -1)
-            .put("partitionKey", true)
+            .put("columnType", PARTITION_KEY.toString())
```

Comment:
Validate `clientId` and `typeSignature` as well. I understand this is not directly related to your pull request. But this is a simple enough change.